addons/point_of_sale/models/pos_order.py

```diff
@@ -540,6 +540,7 @@ def _prepare_balancing_line_vals(self, imbalance_amount, move, session):
             # be the conversion amount for the day the session is being closed.
             imb_amount_session_currency = cur_company._convert(abs(imbalance_amount), cur, session.company_id, fields.Date.context_today(self))
```

Comment:
Hi @yelizariev. Thank you so much for this fix. But what if instead of adding new field in the pos.session, we just modify _this_ line to get the company thru the config_id, like so: `session.config_id.company_id`.
This is because adding the field might not be approved (even it's a related field). Otherwise, I'm fully okay with this fix. :)

Reply:
Should be fine for related field

@jpp-odoo what do you think?

Reply:
@caburj @yelizariev 
In this case, as it's a related field, it's by default non stored, so I don't see why we cannot merge this fix.

Reply:
Okay. Then there is no problem on my side.